apps/frontend/src/App.tsx

```diff
@@ -10,10 +10,12 @@ import Messages from "./pages/Messages";
 import MyReviews from "./pages/MyReviews";
 import MyListings from "./pages/MyListings";
 import MyProfile from "./pages/MyProfile";
+import SearchBar from "./components/searchBar.tsx";
 
 function App() {
   return (
     <Router>
+      <SearchBar />
```

Comment:
I think we should use this component separately on the pages otherwise it's going to break the UI.

I don't think we want the search bar on the Login or Create pages.
<img width="1448" alt="Screenshot 2024-06-22 at 4 05 44 PM" src="https://github.com/UVicMartletplace/martletplace/assets/72957712/6817624e-b5ad-4b2b-8c67-e20c7d86e02a">

Reply:
Updated to be on the pages instead and changed the styling of the account page sidebar to work with the search bar